<launch>
    <!-- set logging level -->
    <param name="log_level" value="0" type="int"/> <!-- DEBUG=0, INFO=1, WARN=2, ERROR=3, FATAL=4 -->
    <env name="ROSCONSOLE_CONFIG_FILE" value="$(find milo)/config/custom_rosconsole.conf" />

    <!-- remap topic namespace -->
    <!-- <remap from="/joystick/cmd_vel" to="/base_controller/cmd_vel" /> -->


    <!-- joy node -->
    <node name="joy_node" pkg="joy" type="joy_node" respawn="true">
        <param name="dev" type="string" value="/dev/input/js1" />
        <param name="deadzone" value="0.25" />
        <param name="autorepeat_rate" value="1" /> <!-- in Hz -->
    </node>

    <!-- teleop node -->
    <node name="milo_teleop_joy" pkg="milo" type="milo_teleop_joy" output="screen" respawn="true">
        <param name="axis_linear" value="1" type=   "int" />
        <param name="axis_angular" value="2" type="int" />
        <param name="scale_linear" value="1.0" type="double" />
        <param name="scale_angular" value="1.0" type="double" />
        <param name="loop_hz" value="100.0" type="double" /> -->
    </node>
    <!-- mbed hardware interface -->
    <node name="milo_hardware_interface" pkg="milo" type="milo_hardware_interface" output="screen" respawn="true">
        <param name="port_mbed" type="string" value="/dev/ttyACM0" />
        <param name="baud_rate" type="int" value="115200" /> -->
        <param name="max_l_speed" type="double" value="1.0" /> -->
        <param name="min_l_speed" type="double" value="-0.5" /> -->
        <param name="max_ang_speed" type="double" value="3.1415" /> -->
        <param name="min_ang_speed" type="double" value="-3.1415" /> -->
        <param name="max_accel" type="double" value="1.0" /> -->
        <param name="min_accel" type="double" value="-1.0" /> -->
        <param name="wheel_radius" type="double" value="0.0325" />  --> 
        <param name="wheel_distance" type="double" value="0.22" /> -->
        <param name="loop_hz" type="double" value="100.0" /> -->
        <!-- <param name="port_mbed" type="string" value="/dev/tnt0" /> -->
    </node>


</launch>
